sdk/core/azure-core-tracing-opencensus/README.md

```diff
@@ -54,7 +54,7 @@ Netty and include OkHTTP client in your pom.xml.
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-core-http-okhttp</artifactId>
-  <version>1.0.0</version>
+  <version>1.1.0-beta.1</version>
```

Comment:
Need to validate the version here

Reply:
yup, I think we just merged Hemant's dec release updates and will include it here.

Reply:
Need to base it on what is going to be released this sprint, after OkHttp is released (or not) this won't point to a package in Maven which will lead to this example failing. Given that OpenTelemetry writes samples with other packages we may need to alter how they are written.

Reply:
We are not releasing OKHttp becausethere is no change since last month.

Reply:
reverted back to 1.0.0 for ok-http